app/main/views/audits.py

```diff
@@ -180,6 +212,48 @@ def acknowledge_audit(audit_id):
     return jsonify(auditEvents=audit_event.serialize()), 200
 
 
+@main.route('/audit-events/<int:audit_id>/acknowledge-including-previous', methods=['POST'])
```

Comment:
With `/audit-events/acknowledge-including-previous/<audit_event_id>` there is the whole naughtiness of having `acknowledge-including-previous` occupying the space that is in other cases occupied by an id though...